utils/build/docker/nodejs/express4/iast/sources.js

```diff
@@ -101,6 +114,82 @@ function init (app, tracer) {
     }
     res.send('OK')
   })
+
+  app.get('/iast/source/kafkavalue/test', (req, res) => {
+    const kafka = getKafka()
+
+    const producer = kafka.producer()
+    const doKafkaOperations = async () => {
+      await producer.connect()
+      await producer.send({
+        topic: 'iast-system-tests-queue',
```

Comment:
How are you creating the topic? the only one I could find in system-tests is this [one](https://github.com/DataDog/system-tests/blob/main/utils/_context/containers.py#L591)

Reply:
ummm, it's created automatically. But if it is a problem we could use `dsm-system-tests-queue`